core/src/main/java/org/apache/iceberg/hadoop/SerializableConfiguration.java

```diff
@@ -26,13 +26,18 @@
 
 /** Wraps a {@link Configuration} object in a {@link Serializable} layer. */
 public class SerializableConfiguration implements Serializable {
+  private static final long serialVersionUID = -8840976521081151175L;
```

Comment:
How is this calculated? Will this break reading old serialized configurations?

Reply:
This is the old serialization ID, otherwise the rev-api check will trigger.